src/test/java/com/wikia/webdriver/common/dataprovider/ads/AdsDataProvider.java

```diff
@@ -190,21 +190,21 @@ private AdsDataProvider() {
             "AAAAAA",
             "FFFFFF"
         }, {
-            "adtest", "Skin",
+            "project43", "SyntheticTests/Skin",
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "SyntheticTests/Skin" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS1192)